aws_lambda_builders/workflows/rust_cargo/workflow.py

```diff
@@ -0,0 +1,59 @@
+"""
+Rust Cargo Workflow
+"""
+
+from aws_lambda_builders.utils import which
+from aws_lambda_builders.path_resolver import PathResolver
+from aws_lambda_builders.workflow import BaseWorkflow, Capability, BuildInSourceSupport, BuildDirectory
+from .actions import RustCargoLambdaBuildAction, RustCopyAndRenameAction
+from .cargo_lambda import SubprocessCargoLambda
+from .exceptions import CargoLambdaExecutionException
+from .feature_flag import is_experimental_cargo_lambda_scope
+
+
+class RustCargoLambdaWorkflow(BaseWorkflow):
+    NAME = "RustCargoLambdaBuilder"
+
+    CAPABILITY = Capability(language="rust", dependency_manager="cargo", application_framework=None)
+
+    DEFAULT_BUILD_DIR = BuildDirectory.SOURCE
+    BUILD_IN_SOURCE_SUPPORT = BuildInSourceSupport.EXCLUSIVELY_SUPPORTED
+
+    SUPPORTED_MANIFESTS = ["Cargo.toml"]
+
+    def __init__(self, source_dir, artifacts_dir, scratch_dir, manifest_path, runtime=None, mode=None, **kwargs):
+        super(RustCargoLambdaWorkflow, self).__init__(
+            source_dir, artifacts_dir, scratch_dir, manifest_path, runtime=runtime, **kwargs
+        )
+        if not is_experimental_cargo_lambda_scope(self.experimental_flags):
+            raise CargoLambdaExecutionException(
+                message="Feature flag `experimentalCargoLambda` must be enabled to use this workflow"
+            )
+
+        # we utilize the handler identifier to
+        # select the binary to build
+        options = kwargs.get("options") or {}
+        handler = options.get("artifact_executable_name", None)
+        flags = options.get("cargo_lambda_flags", None)
```

Comment:
nit: `get` would return `None` if key doesn't exist in the `dict`.
```suggestion
        handler = options.get("artifact_executable_name")
        flags = options.get("cargo_lambda_flags")
```